projects/igniteui-angular/src/lib/directives/mask/mask.directive.ts

```diff
@@ -198,6 +220,34 @@ export class IgxMaskDirective implements OnInit, AfterViewChecked, ControlValueA
          * the end user will be unable to blur the input.
          * https://stackoverflow.com/questions/21406138/input-event-triggered-on-internet-explorer-when-placeholder-changed
          */
+
+        if (this._composing) {
+            if (this.inputValue.length < this._oldText.length) {
+                // software keyboard input delete
+                this._key = this.platform.KEYMAP.BACKSPACE;
+            }
+            return;
+        }
+
+        // After the compositionend event Chromium triggers input events of type 'deleteContentBackward' and
+        // we need to adjust the start and end indexes to include mask literals
+        if (event.inputType === 'deleteContentBackward' && this._key !== this.platform.KEYMAP.BACKSPACE ) {
+            let numberOfMaskLiterals = 0;
+            const literalPos = this.maskParser.getMaskLiterals(this.maskOptions.format).keys();
+            for (const index of literalPos) {
+                if (index >= this.selectionEnd && index < this._end) {
+                    numberOfMaskLiterals++;
```

Comment:
If we have consecutive literals, this doesn't seem to work properly. Maybe we should increment `numberOfMaskliterals` based on how many consecutive literals we encounter?

![maskffs](https://user-images.githubusercontent.com/16020256/124613468-edfc2300-de7b-11eb-9a6e-7f1e0ccca905.gif)